members/user-schema.json

```
@@ -0,0 +1,85 @@
+{
+  "$schema": "https://json-schema.org/draft-04/schema",
+  "$id": "https://github.com/torontojs/members/tree/main/members/user-schema.json",
+  "title": "Members of TorontoJS",
+  "description": "TorontoJS Members",
+  "type": "object",
+  "properties": {
+   "name": {
+      "type": "string",
+        "description": "Name of the user. Full name or internet handle, however you like."
+    },
+    "pronouns": {
+      "type": "string",
+      "description": "Pronouns of the user"
+    },
+    "website": {
+      "type": "string",
+      "description": "Website of the TJS Member"
```

```suggestion
    "website": {
      "type": "string",
      "description": "Website of the TJS Member",
      "format": "uri"
```

I think this will work to have a specific type for URLs.

https://json-schema.org/understanding-json-schema/reference/string#resource-identifiers